app-catalog/src/components/charts/List.tsx

```diff
@@ -140,14 +138,41 @@ export function ChartsList({ fetchCharts = fetchChartsFromArtifact }) {
     );
   }
 
+  function SettingsLink() {
+    const theme = useTheme();
+
+    return (
+      <RouterLink routeName="/settings/plugins/app-catalog" tooltip="App-Catalog Settings">
+        <Typography
+          sx={{
+            size: '1rem',
+            marginLeft: '3rem',
+            color: theme.palette.text.primary,
+          }}
+        >
+          Settings
+        </Typography>
+      </RouterLink>
+    );
+  }
+
+  function TitleOptions() {
+    return (
+      <Box display="flex" justifyContent="space-between" alignItems="center">
+        <Typography variant="h4">Applications</Typography>
```

Comment:
Please check if this plugin has any extra react-axe a11y errors in the developer console.

I expect the heading are not correct here. Style it like a h4, but use the correct semantic heading. So the levels for the page/component are correct.

Reply:
Changed to style and no errors in the dev inspect